style.css

```
@@ -0,0 +1,483 @@
+* {
+    margin: 0;
+    padding: 0;
+}
+
+html, body {
+    height: 100vh;
+}
+
+body {
+    background-color: #1E1E28;
+    margin: 0;
+    padding: 0;
+    font-family: 'Poppins', sans-serif;
+}
+
+#header-block {
+    box-shadow: 1px 1px 1px 1px rgba(0, 0, 0, 0.1);
+    position: fixed;
+    width: 300px;
```

Ao inves de usar px, voce pode usar propriedades dinamicas para te ajudar quando tornar o seu portfolio responsivo. (Isso evita voce de usar media queries)

Este comentario e util para todos os lugares que esta usando um px fixo.

```suggestion
    width: 30em; 
```

ou

```suggestion
    width: 30rem; 
```

---

ou porcentagem. Isso varia de acordo com a necessidade.

---

Certo, feito. thankssssss